packages/js/customer-effort-score/src/customer-feedback-simple/customer-feedback-simple.tsx

```diff
@@ -0,0 +1,138 @@
+/**
+ * External dependencies
+ */
+import {
+	createElement,
+	Fragment,
+	useEffect,
+	useState,
+} from '@wordpress/element';
+import PropTypes from 'prop-types';
+import { Button, Tooltip } from '@wordpress/components';
+import { Text } from '@woocommerce/experimental';
+import { __ } from '@wordpress/i18n';
+
+type CustomerFeedbackSimpleProps = {
+	recordScoreCallback: ( score: number ) => void;
+	label: string;
+	feedbackScore?: number;
+	showFeedback?: boolean;
+};
+
+/**
+ * Provides a modal requesting customer feedback.
+ *
+ * A label is displayed in the modal asking the customer to score the
+ * difficulty completing a task. A group of radio buttons, styled with
+ * emoji facial expressions, are used to provide a score between 1 and 5.
+ *
+ * A low score triggers a comments field to appear.
+ *
+ * Upon completion, the score and comments is sent to a callback function.
+ *
+ * @param {Object}   props                     Component props.
+ * @param {Function} props.recordScoreCallback Function to call when the results are sent.
+ * @param {string}   props.label               Question to ask the customer.
+ * @param {number}   props.feedbackScore       Feedback score.
+ * @param {boolean}  props.showFeedback        Show feedback.
+ */
+const CustomerFeedbackSimple: React.FC< CustomerFeedbackSimpleProps > = ( {
+	recordScoreCallback,
+	label,
+	feedbackScore = NaN,
```

Comment:
Is this `initialScore`?

Reply:
No, it is the actual score, allowing the parent component to override the score within this components scope.

Reply:
So the parent component provides a score, but the score is also housed in state?  What's the use case for allowing the parent to override?

Reply:
The main use case was when a user submits a score that triggers the modal ( anything less then Neutral) then closes the Modal without submitting the score. 
We don't want to show the 'Thank you for your feedback...' text and un-select the selected score then but keep the CES displayed. Which was the main reason for resetting it, and allowing that to be done by the parent component.